environment.yml

```diff
@@ -0,0 +1,26 @@
+name: py2_parcels
+channels:
+  - https://conda.anaconda.org/conda-forge
+  - https://repo.continuum.io/pkgs/free
```

Comment:
The channels field can be reduced to 
```yaml
channels:
- conda-forge
```

The second line is the default fall-back channel for anything not found in explicitly listed channels anyway. And the explicit URLs might change in the future while the short-form `conda-forge` will just refer to whatever is the current conda-forge-repo URL. (I tend to put full URLs there, because I have some setups behind firewalls where I have to point this to a mirror.)